device_doctor/tool/build.bat

```diff
@@ -1,8 +1,8 @@
-@ECHO off
-REM Copyright 2020 The Flutter Authors. All rights reserved.
-REM Use of this source code is governed by a BSD-style license that can be
-REM found in the LICENSE file.
+:: Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
Why do we require scripts?

Reply:
You mean why we have build.bat/build.sh? They are used to build the tool.